### ISSUE TITLE
PHP Support

### ISSUE THREAD

**Issue by SamMousa** [dabilite/yii2-user#15]
I think we should drop support for all non current versions of PHP.

http://php.net/supported-versions.php

@tsdogs  @thyseus what do you think?

Dropping support for me means:
1. We remove the php version from travis tests.
2. We are allowed to use newer language features.

**Comment by thyseus** [comment]
I think we should lean to the version constraints that the Yii Framework pretends. 

Which is >PHP 5.4 actually and may be risen to > PHP 5.6 for Yii 2.1.0 iirc.

**Comment by tsdogs** [comment]
Yes stick to what Yii2 Framework does. (I had an error on UserSearch.php line 96 if I remember correctly) on php 5.6 on debian)
Lately I also had some troubles with phpunit on php 5.6 they seem to have dropped support for the php 5 series...

**Comment by thyseus** [comment]
I also encountered the problem with phpunit and 5.6. I really wonder why yii calls code inside phpunit on every request, even when not in debug mode. I did not find the time to investigate this yet.

**Comment by tsdogs** [comment]
I solved it by using a more recent composer.json from yii2 app templates (using no-dev though)
But at the end updated to 7.0 as it's the one I'm using on dev machine.

**Comment by tsdogs** [comment]
Please merge this: https://github.com/dektrium/yii2-user/pull/933

**Comment by SamMousa** [comment]
Will do tomorrow
